README.md

```diff
@@ -29,6 +29,25 @@ The Miniflux documentation is available here: <https://docs.miniflux.net/>
 - [Upgrading to a new version](https://docs.miniflux.net/en/latest/upgrade.html)
 - [Configuration](https://docs.miniflux.net/en/latest/configuration.html)
 
+Running local (development) instance
```

Comment:
There is already a developer documentation here: https://docs.miniflux.net/en/latest/development.html

I don't think this part should be part of the readme. By the way, you don't need Docker to use Postgres locally. You can use Docker if you want but it's not a requirement.

Reply:
Ok, I will remove that part from the docs.